src/wwwroot/app/services/UserDialogService/UserDialogService.js

```diff
@@ -0,0 +1,83 @@
+import _dialog from './dialog.view.html';
+import './dialog.scss';
+
+import {
+   assign,
+   invoke
+} from 'lodash';
+
+let _$q, _ModalFactory, _NotificationFactory, _$translate;
+
+export default class UserDialogService {
+
+   constructor($q, $translate, ModalFactory, NotificationFactory) {
+      'ngInject';
+      _$q                  = $q;
+      _ModalFactory        = ModalFactory;
+      _NotificationFactory = NotificationFactory;
+      _$translate          = $translate;
+   }
+
+   confirm(question) {
+      let deferred = _$q.defer();
+
+      let contentScope = {
+         ok     : deferred.resolve,
+         cancel : deferred.reject
+      };
+
+      let buttons = [{ name: _$translate.instant('COMMON.OK'),     func: 'ok' },
```

Comment:
i think will be much better if u use function instead of funcName
`{ name: _$translate.instant('COMMON.OK'),     func: deferred.resolve }`

and don't use `contentScope` in this case